### PR TITLE
Escape html templates in app manager

### PR DESCRIPTION
**Pull request by AmitPhulera**
## Summary

https://dimagi-dev.atlassian.net/browse/SAAS-11428

This is one of the PRs from the series of PRs that will be replacing `<%=` to `<%-`.

This PR specifically covers the changes of App Manager.

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### QA Plan

QA plan is described here https://dimagi-dev.atlassian.net/browse/QA-2854?focusedCommentId=133830 
### Safety story

The local testing and notes have been updated in the doc https://docs.google.com/spreadsheets/d/1oBLABIn41A7LbslViK997GmaQsQ5kRUJNDyn9WzFpGc/edit#gid=0
### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: corehq/apps/app_manager/static/app_manager/js/summary/models.js [hunk 1]
```diff
@@ -298,8 +298,8 @@ hqDefine('app_manager/js/summary/models',[
         self.moduleFormReference = function (formId) {
             var formData = self.formNameMap[formId];
             var template = self.readOnly
-                ? "<%= moduleName %> &rarr; <%= formName %>"
-                : "<a href='<%= moduleUrl %>'><%= moduleName %></a> &rarr; <a href='<%= formUrl %>'><%= formName %></a>"
+                ? "<%- moduleName %> &rarr; <%- formName %>"
```
Comment (orangejenny) [inline corehq/apps/app_manager/static/app_manager/js/summary/models.js:301]: Confirming that if a module or form name had, say, a `<` in it, it'll still display the same as it did previously?

**Review by orangejenny — APPROVED** [review]
Left a question, but if the answer to that is "there won't be any user-facing changes" this looks good.

**Review by dannyroberts — APPROVED** [review]

File: corehq/apps/app_manager/static/app_manager/js/summary/models.js (hunk 1 above)
Comment (AmitPhulera) [reply corehq/apps/app_manager/static/app_manager/js/summary/models.js:301]: Yep, it does. Here is an example https://staging.commcarehq.org/a/ap-test-project/apps/view/70920c6ee9136f4fc99dd5ca57f5cfe9/summary/case/

Comment (orangejenny) [reply corehq/apps/app_manager/static/app_manager/js/summary/models.js:301]: sweet, thank you

**Pull request merged**